fbw-a380x/src/systems/extras-host/modules/telex_check/TelexCheck.ts

```diff
@@ -0,0 +1,27 @@
+// If the consent is not set, show telex page
+
+// Copyright (c) 2024 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import { NXDataStore, PopUpDialog } from '@flybywiresim/fbw-sdk';
+
+export class TelexCheck {
+  constructor() {
+    console.log('TelexCheck: Created');
+  }
+
+  public startPublish(): void {
```

Comment:
It's not a publisher so this seems weird.

Reply:
Yes 😄 will change it